cmd/neofs-node/netmap.go

```diff
@@ -29,6 +50,11 @@ func initNetmapService(c *cfg) {
 			),
 		),
 	)
+
+	c.cfgNetmap.state = newNetworkState()
+	addNewEpochNotificationHandler(c, func(ev event.Event) {
+		c.cfgNetmap.state.setCurrentEpoch(ev.(netmapEvent.NewEpoch).EpochNumber())
+	})
```

Comment:
One more thing: update network state with epoch value from sidechain at `bootstrapNode()` function. This way node will be ready to create objects right from the start.

Reply:
What component and function should I use in `bootstrapNode`?

Reply:
I think network map wrapper missing `Epoch` method implementation to get current epoch value. Fix it and then invoke this method in `bootstrapNode()`. 

```go
epoch := c.cfgNetmap.wrapper.Epoch()
c.cfgNetmap.state.setCurrentEpoch(epoch)
```